dcos_installer/backend.py

```diff
@@ -45,13 +39,13 @@ def do_configure(config_path=CONFIG_PATH):
     """
     config = Config(config_path)
 
-    validation = validate_gen(config)
-
-    if not validation:
+    try:
+        config_util.do_configure(config)
```

Comment:
❓ Separate really but it looks like `config_util.do_configure` is only called here. If that's right, what do you think of removing the function for consistency with https://github.com/dcos/dcos/pull/1902/files#diff-d66881dc86f4e8947b79ac9c556d0fe1R60 ?